### ISSUE TITLE
Cody: VSCode history should list the current chat

### ISSUE THREAD

**Issue by toolmantim** [sourcegraph/sourcegraph#52544]
Now we have the [➕] button instead of clear conversation, we need to ensure the current conversation is also at the top of the conversation history list. Because at the moment, the [➕] button is more like a "Clear conversation" function vs the ChatGPT style list of conversations (where the current one is highlighted).

#### Steps to reproduce:

1. Click history, and it's empty, but should list the current conversation

**Comment by toolmantim** [comment]
Is this a tricky one to do @abeatrix?

**Comment by abeatrix** [comment]
@toolmantim Would this work as a temporary solution?


![Image](https://github.com/sourcegraph/sourcegraph/assets/68532117/70e9f51e-4d88-4d2a-b9e1-bc5ff9fe6800)

**Comment by toolmantim** [comment]
Yesssss, that's pretty much it @abeatrix! 😍 And I can clean up the labelling.

**Comment by abeatrix** [comment]
@toolmantim Perfect! I've created a PR draft, feel free to play around with the labels in the PR and let me know if there are other things you would like me to change :) 
https://github.com/sourcegraph/sourcegraph/pull/52672

**Comment by abeatrix** [comment]
Side notes, do you think we should use the first request from human as the description for each history item? I found using the last reply from Cody quite unhelpful lol

![image](https://github.com/sourcegraph/sourcegraph/assets/68532117/0251cabd-9d9d-4c4c-9ce7-76636e114688)

**Comment by toolmantim** [comment]
Yeah I found that too @abeatrix! Probably the last human request might be best?

An out there idea: could we ask Cody to come up with a good summary/title for the convo ala ChatGPT?

I'd love us to use friendly/relative dates too.

**Comment by abeatrix** [comment]
Update: Covered by https://github.com/sourcegraph/sourcegraph/pull/52904

**Comment by toolmantim** [comment]
@deepak2431 I've noticed that sadly the chat history doesn't contain the current chat at the top still (like a list of conversations, current one included and highlighted). Was it intentional to leave it off, or has there been a regression?

**Comment by deepak2431** [comment]
@toolmantim The current chat will be added to the history only after at least one conversation.  I have implemented this behaviour so that it won't have a lot of blank items in the history chat. So, it was intentional to implement this way.
